src/session.ts

```diff
@@ -269,10 +269,12 @@ export function openExternalBrowser(): void {
 
 export async function showWebView(file: string, title: string, viewer: string | boolean): Promise<void> {
     console.info(`[showWebView] file: ${file}, viewer: ${viewer.toString()}`);
+    console.log(file);
```

Comment:
Is this redundant since the line above just prints `file` to console?

Reply:
:facepalm: thanks for catching that